code/src/reducers/questions.js

```diff
@@ -0,0 +1,66 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+ const questions = createSlice({
+	name: 'questions',
+	initialState: {
+			userName: null,
+			gameStatus: undefined,
+      		loading: false,
+          history: []
+			
+	},
+	reducers: {
+			setUserName: (store, action) => {
```

Comment:
another quick look at the indentation could make it easier to read ✨

Reply:
TAB bonanza 🤠 🤠